static/app/views/performance/traces/fieldRenderers.tsx

```diff
@@ -170,26 +178,34 @@ export function SpanBreakdownSliceRenderer({
   trace: TraceResult<Field>;
   offset?: number;
   sliceDurationReal?: number;
+  sliceNumberStart?: number;
+  sliceNumberWidth?: number;
 }) {
-  const traceSliceSize = (trace.end - trace.start) / BREAKDOWN_NUM_SLICES;
-  const traceDuration = BREAKDOWN_NUM_SLICES * traceSliceSize;
+  const traceDuration = trace.end - trace.start;
 
   const sliceDuration = sliceEnd - sliceStart;
 
-  if (sliceDuration <= 0) {
+  if (sliceNumberWidth === 0 && sliceNumberWidth === undefined && sliceDuration <= 0) {
```

Comment:
`sliceNumberWidth === 0 && sliceNumberWidth === undefined`

Isn't that always false?

Reply:
This should just be entirely removed anyway actually, we have checks on the backend now, don't need this defensive code anymore.